src/plugins/packages/themes.js

```diff
@@ -46,7 +46,7 @@ function makeLocalThemeObject(css) {
 	const themeName = css.slice(0, -4);
 	return {
 		displayName: themeName.charAt(0).toUpperCase() + themeName.slice(1),
-		filename: path.join(__dirname, "..", "..", "public", "themes", `${themeName}.css`),
+		filename: path.join(__dirname, "..", "..", "..", "public", "themes", `${themeName}.css`),
```

Comment:
Note that this line is currently not being used because of https://github.com/thelounge/lounge/issues/1969.